Source/Creation/Frame/Content/String/ID3CreditsListFrameCreator.swift

```diff
@@ -29,28 +29,30 @@ class ID3CreditsListFrameCreator: CreditsListFrameCreator {
     ) -> [UInt8] {
         let frameContents = [UInt8](
             [
-                entries: [(
-                    stringToBytesAdapter.adapt(
-                        string: role,
-                        for: version,
-                        includingEncoding: true,
-                        includingTermination: true
-                    ),
-                    stringToBytesAdapter.adapt(
-                        string: person,
-                        for: version,
-                        includingEncoding: false,
-                        includingTermination: true
-                    )
-                    )]
+                stringToBytesAdapter.encoding(for: version),
```

Comment:
I've been trying to figure this out for hours, but I can't work out how to write this function:

```swift
    func createFrame(
        frameIdentifier: [UInt8],
        version: ID3Version,
        entries: [(role: String, person: String)]
    ) -> [UInt8] {
        let frameContents = [UInt8](
            [
                stringToBytesAdapter.encoding(for: version),
                entries: [
                stringToBytesAdapter.adapt(
                string: role,
                for: version,
                includingEncoding: true,
                includingTermination: true
                ),
                stringToBytesAdapter.adapt(
                string: person,
                for: version,
                includingEncoding: false,
                includingTermination: true
                )
                ]
                ].joined()
        )
        
        return [UInt8](
            [
                frameIdentifier,
                frameContentSizeCalculator.calculateSizeOf(
                    content: frameContents,
                    version: version),
                frameFlagsCreator.createFor(version: version),
                frameContents
                ].joined()
        )
    }
```